src/com/CommunicateMesh.hpp

```diff
@@ -31,12 +31,12 @@ class CommunicateMesh {
       mesh::Mesh &mesh);
 
   void sendBoundingBox(
-      const mesh::Mesh::BoundingBox &bb,
-      int                            rankReceiver);
+      const mesh::BoundingBox &bb,
+      int                     rankReceiver);
 
   void receiveBoundingBox(
-      mesh::Mesh::BoundingBox &bb,
-      int                      rankSender);
+      mesh::BoundingBox &bb,
+      int               rankSender);
```

Comment:
Why are both these functions needed at all? Couldn't they be replaced with calls to the identical functions of `CommunicateBoundingBox`?